CHANGELOG.md

```diff
@@ -3,35 +3,92 @@
-* Template update to nf-core tools v1.12.1
 * [#85](https://github.com/nf-core/bcellmagic/pull/85): Added support for TCR data with params: `--loci`
 * [#85](https://github.com/nf-core/bcellmagic/pull/85): Added support for mice data and possibility for other species with params: `--species`
-* [#85](https://github.com/nf-core/bcellmagic/pull/85): Added support for 5' RACE technology with params: `--race_5prime`, `--race_linker`
+* [#85](https://github.com/nf-core/bcellmagic/pull/85): Added support for 5' RACE technology with params: `--protocol`, `--race_linker`
 * [#87](https://github.com/nf-core/bcellmagic/pull/87): Added tests for TCR data
+* [#102](https://github.com/nf-core/bcellmagic/pull/102): Added param `--protocol`.
+* [#102](https://github.com/nf-core/bcellmagic/pull/102): Added support for C-primer in any R1 or R2 with param `--cprimer_position`.
+* [#102](https://github.com/nf-core/bcellmagic/pull/102): Bump versions to 2.0 and NXF 21.04.0.
+* [#103](https://github.com/nf-core/bcellmagic/pull/103): Added full size tests (pcr_umi).
+* [#114](https://github.com/nf-core/bcellmagic/pull/114): Added parameter `--skip_lineages`.
+* [#112](https://github.com/nf-core/bcellmagic/pull/112): Template update to nf-core tools v1.14.
+* [#114](https://github.com/nf-core/bcellmagic/pull/114): Added Bcellmagic html report.
+* [#114](https://github.com/nf-core/bcellmagic/pull/114): Improved documentation on amplicon protocol support.
+* [#115](https://github.com/nf-core/bcellmagic/pull/115): Improved output file structure and documentation.
 
 ### `Fixed`
 
 * [#74](https://github.com/nf-core/bcellmagic/pull/74): Fixed AWStest workflow
 * [#75](https://github.com/nf-core/bcellmagic/pull/75): Fixed lineage trees publish plots and graphml
 * [#75](https://github.com/nf-core/bcellmagic/pull/75): Assemble process shorten filename to avoid error due to too long filename
 * [#87](https://github.com/nf-core/bcellmagic/pull/87): Order sequence logs by sample ID.
+* [#104](https://github.com/nf-core/bcellmagic/pull/104): Fix bug in pairseq barcode copy before consensus.
+* [#114](https://github.com/nf-core/bcellmagic/pull/114): Analysis not restricted to Ig heavy chains.
 
 ### `Dependencies`
 
+Note, since the pipeline is now using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.
+
+| Dependency   | Old version | New version |
+|--------------|-------------|-------------|
+| Python       | v3.6.11     | 3.8.0,3.0.3 |
+| markdown     | v3.1.1      |             |
+| muscle       | v3.8.1551   |             |
+| vsearch      | v2.11.1     |             |
+| fastqc       | 0.11.8      | 0.11.9      |
+| multiqc      | 1.9         | 1.9         |
```

Comment:
Could update to latest release here without breaking anything, e.g. multiqc == 1.11